doc/settings.md

```diff
@@ -1,6 +1,11 @@
 # Ada Language Server configuration
 
-The LSP has a dedicated notification to signal the server the change of configuration settings. Any setting should be inside a `ada` JSON object.
+The ALS reads initial configuration from the `initializationOptions`
+property of the `initialize` request (if any) and then updates
+the configuration with each `workspace/didChangeConfiguration`
+notification. Any setting in `workspace/didChangeConfiguration` should
+be inside a `ada` JSON object, while there is no such wrapping object
+for `initializationOptions`.
```

Comment:
Maybe let's add a minimal example to explain the difference between dicChange and initialize request options?